docs/module_docs/featdet_adap_chromatogram_builder/adap-chromatogram-builder.md

```diff
@@ -1,33 +1,36 @@
+:material-menu-open: Feature detection → LC-MS → ADAP chromatogram builder
+![ADAP Chromatogram Builder](adap_chromatogram_builder.png)
 
-The scan selection parameter specifies the scans that shall be processed for feature detection.
-Usually, setting the ms level to 1 is sufficient. If a calibration segment is present, it can be cut
-out via the retention time filter in the scan selection.
+#### **Raw data files**
+Select the input raw data files for chromatogram building. Mass lists associated with the data files will be automatically selected. See option descriptions in [Mass detection](../featdet_mass_detection/mass-detection.md#parameters-settings) module.
 
-### Min group size in number of scans
+####  **Scans**
+Select (or filter out) the MS scans to be processed. Several filters are avialble (see option descriptions in [Mass detection](../featdet_mass_detection/mass-detection.md#parameters-settings) module) but setting the _MS level = 1_ is usually sufficient for this module. 
 
-This parameter specifies the number of consecutive detections of the same m/z value in a
-chromatographic peak (rt dimension). This means that a single m/z has to be detected in, e.g, 5
-scans with an intensity higher than zero. This parameter helps to filter noise. Usually no less than
-5 should be set here if the MS1 acquisition rate is sufficient.
+####  **Min group size in # of scans**
+Minimum number of consecutive MS1 scans where a _m/z_ must be detected with a non-zero intensity in order for the corresponding EICs to be considered valid and retained in the feature list.
+:octicons-light-bulb-16: **Tip**. This parameter largely depends on the chromatographic system setup (_e.g._ HPLC vs UHPLC) and the acquisition rate (_a.k.a._ MS scan speed) of the mass spectrometer. The best way to optimize this setting is by manually inspecting the the raw data and determining the typical minimum number of data points of the LC peaks. Usually, no less than 4-5 should be used.
 
-### Group intensity threshold
+#### **Group intensity threshold**
+Minimum signal intensity that the group scans (see previous parameter) must exceed in order for the corresponding EICs to be considered valid and retained in the feature list.
+:octicons-light-bulb-16: **Tip**. A good starting point for this parameter is the same noise level used in the [Mass detection](../featdet_mass_detection/mass-detection.md).
 
-Specifies a minimum intensity that the number specified
-by [min group size](#min-group-size-in-number-of-scans) have to exceed. In this example, the intensity in
-at least 5 scans must be above 5E2.
+#### **Min highest intensity**
+Minimum intensity that the highest point in the EIC must exceed in order for the corresponding trace to be considered valid and retained in the feature list.
+:octicons-light-bulb-16: **Tip**. A good starting point for this parameter is 3 times the noise level used in the [Mass detection](../featdet_mass_detection/mass-detection.md).
```

Comment:
Usually i'd take a higher value here, like 5 to 10 times as much as the noise level, but i also use 2-3 times the noise level for the group intensity threshold. @robinschmid / @titodamiani  what do you think?

Reply:
I was actually thinking about it today while looking at the timsTOF data. I agree with Steffen. I’d say 3 times the noise level is good for group intensity threshold and around 10 is good for the Min highest intensity. But this requires the noise level to be an "instrumental noise", so pretty low. What do you think?